ivadomed/main.py

```diff
@@ -244,7 +252,8 @@ def cmd_train(context):
     var_contrast_list = []
 
     # Loss
-    if context["loss"]["name"] in ["dice", "cross_entropy", "focal", "gdl", "focal_dice"]:
+    if context["loss"]["name"] in ["dice", "cross_entropy", "focal", "gdl", "focal_dice", "multi_class_dice",
+                                   "combined_dice"]:
```

Comment:
cf comment about dice/multiclass/combined

Reply:
Ok made the change, but it requires to add another parameter `dice_params` in the config files. Let me know if that's what you had in mind :)

Reply:
Sorry I am missing something: why for?
I just had in mind to call dice --> loop across a list of one or more elements.